src/editor/utils/elementapimixin.js

```diff
@@ -19,8 +20,17 @@ const ElementApiMixin = {
 	/**
 	 * @inheritDoc
 	 */
-	updateElement() {
-		setDataInElement( this.element, this.data.get() );
+	updateSourceElement() {
+		if ( !this.sourceElement ) {
+			/**
+			 * The `sourceElement` is required by the `ElementApi` interface.
+			 *
+			 * @error elementapimixin-missing-sourceelement
+			 */
+			throw new CKEditorError( 'elementapi-missing-sourceelement: The "sourceElement" is required by the "ElementApi" interface.' );
```

Comment:
How's an external developer supposed to understand this? He/she doesn't know our interfaces. They know what method they used. They should learn that it doesn't work if the editor wasn't created by passing the source element.

PS. Keep the error msg short. Explain it more thoroughly in the comment above.

Reply:
I improved a little the error description but I'm not able to check it out so I'm not sure whether it works... Could you help me? :D